docs/_docs/Instructors/02-Teachers-How-to-build-Learning-Units.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: How to build Learning Units
 category: Instructors
-order: 1
+order: 2
 ---
 
 > ⚠️ Some information in this page might be outdated.
```

Comment:
Reminder: To be removed after page is finished

Reply:
Will be done in a different MR